fbc/catalog-source/README.md

```diff
@@ -0,0 +1,54 @@
+
+* This sequence assumes the use of an index image located at **quay.io/scoheb/fbc-index-testing:latest**
+* It should contain a FBC fragment added by IIB
+
+
+**login as kubeadmin**
+
+```
+oc login ...
+```
+
+**create new namespace**
+```
+oc new-project my-namespace
+```
+
+**disable other catalog sources**
+
+```
+sh ./patch.sh
+```
+
+**create catalog source**
+
+```
+oc apply -f catalogsource.yaml
+```
+
+**create operator group**
+
+```
+oc apply -f operatorgroup.yaml
+```
+
+**create subscription**
+
+```
+oc apply -f subscription.yaml
+```
+
+**login to console**
+
+**goto Operators/OperatorHub**
+
+**search for 'virtualization'**
+
+**select the one for 'My Operator Catalog'**
+
+**click 'Install'**
+
+**click 'Install'**
```

Comment:
is this duplicated for any reason?

Reply:
I encountered 2 separate Install buttons during the flow

Reply:
Maybe you could change it to `Automatic` in the subscription. I changed mine and it works. So you can remove these two lines.